database_migrations/versions/20210209_134758_rename_enumtables.py

```diff
@@ -0,0 +1,264 @@
+        sa.Column("item_id", sa.String(), nullable=False),
+        sa.PrimaryKeyConstraint("item_id", name=op.f("pk_entity_types")),
+        schema="aspen",
+    )
+    op.create_table(
+        "public_repository_types",
+        sa.Column("item_id", sa.String(), nullable=False),
+        sa.PrimaryKeyConstraint("item_id", name=op.f("pk_public_repository_types")),
+        schema="aspen",
+    )
+    op.create_table(
+        "workflow_types",
+        sa.Column("item_id", sa.String(), nullable=False),
+        sa.PrimaryKeyConstraint("item_id", name=op.f("pk_workflow_types")),
+        schema="aspen",
+    )
+    op.enum_insert(
+        "entity_types",
+        [
+            "HOST_FILTERED_SEQUENCE",
+            "BAM",
+            "SEQUENCING_READS",
+            "PROCESSED_GISAID_DUMP",
+            "PATHOGEN_GENOME",
+            "RAW_GISAID_DUMP",
+        ],
+        schema="aspen",
+    )
+    op.enum_insert("public_repository_types", ["GISAID", "NCBI_SRA"], schema="aspen")
+    op.enum_insert("data_types", ["SEQUENCES", "TREES", "METADATA"], schema="aspen")
+    op.enum_insert(
+        "workflow_types",
+        [
+            "ALIGN_READ",
+            "CALL_CONSENSUS",
+            "PHYLO_RUN",
+            "FILTER_READ",
+            "PROCESS_GISAID_DUMP",
+        ],
+        schema="aspen",
+    )
+
+    ####################################################################################
+    # Move the FK to the new types tables.
+    op.drop_constraint(
+        "fk_can_see_data_type_datatypes",
+        "can_see",
+        schema="aspen",
+        type_="foreignkey",
+    )
+    op.create_foreign_key(
+        op.f("fk_can_see_data_type_data_types"),
+        "can_see",
+        "data_types",
+        ["data_type"],
+        ["item_id"],
+        source_schema="aspen",
+        referent_schema="aspen",
+    )
+    op.drop_constraint(
+        "fk_entities_entity_type_entitytypes",
+        "entities",
+        schema="aspen",
+        type_="foreignkey",
+    )
+    op.create_foreign_key(
+        op.f("fk_entities_entity_type_entity_types"),
+        "entities",
+        "entity_types",
+        ["entity_type"],
+        ["item_id"],
+        source_schema="aspen",
+        referent_schema="aspen",
+    )
+    op.drop_constraint(
+        "fk_public_repository_entity_type_publicrepositorytypes",
+        "public_repository",
+        schema="aspen",
+        type_="foreignkey",
+    )
+    op.create_foreign_key(
+        op.f("fk_public_repository_entity_type_public_repository_types"),
+        "public_repository",
+        "public_repository_types",
+        ["entity_type"],
+        ["item_id"],
+        source_schema="aspen",
+        referent_schema="aspen",
+    )
+    op.drop_constraint(
+        "fk_workflows_workflow_type_workflowtypes",
+        "workflows",
+        schema="aspen",
+        type_="foreignkey",
+    )
+    op.create_foreign_key(
+        op.f("fk_workflows_workflow_type_workflow_types"),
```

Comment:
this is a really confusing/ redundant name as well

Reply:
is it common practice to name foreign keys like this? Maybe I'm unfamiliar with how these are usually named.